services/motion/builtin/builtin_test.go

```diff
@@ -639,8 +645,8 @@ func TestStoppableMoveFunctions(t *testing.T) {
 			motionCfg := motion.MotionConfiguration{
 				PlanDeviationMM:       10000,
 				LinearMPerSec:         10,
-				PositionPollingFreqHz: 4,
-				ObstaclePollingFreqHz: 1,
+				PositionPollingFreqHz: &defaultPositionPollingHz,
```

Comment:
If these are defaults, don't they not need to be specified?

Reply:
I might be able to remove them.  I was just doing 1:1 swaps